### ISSUE TITLE
Docker issue: volume name is too short, names should be at least two alphanumeric characters

### ISSUE THREAD

**Issue by rileythai** [sdss/zora_valis_dockers#1]
Unsure of where it originates from -- maybe if the env variables aren't set beforehand? Adding a `/` after the spec seems to fix it.

Issue occurs on both production and development `docker-compose`.

```Creating network "zora_valis_dockers_valisnet" with the default driver
Creating valis ... error

ERROR: for valis  Cannot create container for service valis: create .: volume name is too short, names should be at least two alphanumeric characters

ERROR: for valis  Cannot create container for service valis: create .: volume name is too short, names should be at least two alphanumeric characters
ERROR: Encountered errors while bringing up the project.```

**Comment by havok2063** [comment]
@rileythai Interesting.  I hadn't seen this error before.  what version of docker and docker-compose are you using?

**Comment by rileythai** [comment]
> @rileythai Interesting. I hadn't seen this error before. what version of docker and docker-compose are you using?
```
$ docker --version
> Docker version 24.0.7, build afdd53b
```
When I encountered this error, it was on a different `docker` version. The `docker-compose` still fails on this version with this error:
```
1 error(s) decoding:

* error decoding 'volumes[1]': invalid spec: :/root/sas: empty section between colons
```

The fix in #2 nonetheless resolves this error though.
